layouts/shortcodes/code_block_from.md

```diff
@@ -17,9 +19,8 @@
 {{ end -}}
 {{ $fileContent := readFile $path -}}
 {{ $fileLines := split $fileContent "\n" -}}
-{{ $excerpt := after $from $fileLines -}}
+{{ $excerpt := first (sub (int $to) (int $from)) (after $from $fileLines) -}}
 
 ```{{ $lang }}
-# {{ $file }}
```

Comment:
@chalin I was wondering if this could be removed, since the `#` will only signify a line comment in some languages. I could add an argument to the function that is false by default that could show the filename.

Unrelated, there's also a more feature rich `highlight` command in Hugo which might be worth switching to for some of the cool features lacking in the grave accent surrounded code blocks

Reply:
I've incorporated your changes, and made a few more -- see #1148. Hopefully this addresses your needs for now. As I mention in that PR, I'm purposefully keeping this shortcode simple, making minimal increments. I'm hoping to implement a more complete/powerful code-excerpting solution soon (from another project) as soon as I can.

(Yes, I'd been giving `highlight` some thought.)

Reply:
@mic-max - thanks for the review of #1148. It has been merged, so you can rebase. Let me know if you need more features, or you can just make the changes here (as you've done).